index.js

```diff
@@ -277,6 +277,21 @@
     return Nothing();
   });
 
+  //. pluck :: String -> [Object] -> [Maybe *]
```

Comment:
It would be nice if we could use `String -> [{String: a}] -> [Maybe a]` here, but this isn't accurate since the objects may be "records" with mixed value types (e.g. `{ a :: Number, b :: Boolean }`).